README.md

```diff
@@ -1,13 +1,10 @@
 # Project Name
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This project was to build a chat bot using HTML and Javascript. 
 
 ## The problem
-
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+We were given the base code in HTML and CSS and needed to mostly work in Javascript to get the chat bot going. 
+First I linked all the elements of the HTML with the JS in the DOM section and then tried to get a  very simple chat bot going. I planned more complicated elements but could not achieve them with my level of knowledge and time given. I would have liked the boring vacation button to loop the chat back to the beginning somehow and also to have a dropdown menu of some kind. 
 
 ## View it live
-
-Have you deployed your project somewhere? Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://app.netlify.com/sites/chat-bot-technigo/overview
```

Comment:
The link to the deployed site is not working.

Reply:
Great thanks! I fixed this